pages/examples/blocking-callback-2.py

```diff
@@ -0,0 +1,90 @@
+from dash_extensions.enrich import (
+    DashProxy,
+    dcc,
+    html,
+    Output,
+    Input,
+    State,
+    BlockingCallbackTransform,
+    Trigger,
+    TriggerTransform,
+)
+import plotly.express as px
+
+external_stylesheets = [
+    "https://codepen.io/chriddyp/pen/bWLwgP.css",
+    {
+        "href": "https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css",
+        "rel": "stylesheet",
+        "integrity": "sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO",
+        "crossorigin": "anonymous",
+    },
+]
+
+df = px.data.gapminder()
+years = df.year.unique()
+
+app = DashProxy(
+    __name__,
+    transforms=[BlockingCallbackTransform(timeout=10), TriggerTransform()],
+    external_stylesheets=external_stylesheets,
+)
+
+app.layout = html.Div(
+    [
+        graph := dcc.Graph(),
+        html.Br(),
+        slider := dcc.Slider(
+            min=years.min(),
+            max=years.max(),
+            value=years.max(),
+            step=years[1] - years[0],
+            marks={int(i): str(i) for i in years},
```

Comment:
There might be a bug here, if you pass in `np.int64`, the layout will crash without any error.